test/biometric_storage_test.dart

```diff
@@ -17,7 +18,8 @@ void main() {
   });
 
   tearDown(() {
-    channel.setMockMethodCallHandler(null);
+    TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger
```

Comment:
```suggestion
    TestDefaultBinaryMessengerBinding.instance?.defaultBinaryMessenger
```

a `?` is required for me on dart 2.19.6